modules/internal/ChapelRange.chpl

```diff
@@ -144,7 +144,8 @@ module ChapelRange {
   
     var _low: idxType = 1;                         // lower bound
     var _high: idxType = 0;                        // upper bound
-    var _stride: strType = 1;                      // signed stride of range
+    var _stride: if stridable then strType else void =
+                   if stridable then 1 else _void; // signed stride of range
```

Comment:
This might read more cleanly as:

````chapel
var _stride = if stridable then 1:strType else _void;
````

This is also the first time I noticed that there's a '_void' value.  I'd like to come up with another notation for this that doesn't use an underscore...  (particularly if we expect users to use it).